src/pydata_sphinx_theme/assets/scripts/pydata-sphinx-theme.js

```diff
@@ -236,12 +236,18 @@ var addEventListenerForSearchKeyboard = () => {
     (event) => {
       let input = findSearchInput();
       // toggle on Ctrl+k or ⌘+k
-      if ((event.ctrlKey || event.metaKey) && event.code == "KeyK") {
+      if (
+        (event.ctrlKey || event.metaKey) &&
+        (/k/i.test(event.key) || event.keyCode === 75)
```

Comment:
Since this might seem obscure for folks unfamiliar with non-QWERTY layouts, perhaps a comment like this one would be good
```suggestion
		// Needed for DVORAK key-mapping
        (/k/i.test(event.key) || event.keyCode === 75)
```

Reply:
why not simply "non en-iso" instead of "DVORAK" ? If I understand correctly this fix is for any keyboard mapping right ?

Reply:
I don't think it really needs commenting. Most keyboard shortcuts should match the **letter** (event.key) that would appear in a text editor or input field and **not** the physical **key** (event.code) on your keyboard. I know, I know, the terminology is confusing, but that's the way it is. 

The exception would be if you're really trying to match the shortcut to a specific physical place on the keyboard. For example, imagine if you were trying to create an app that turns your keyboard into a piano-like musical instrument. Then you would want to map the physical location of keys on the keyboard rather than the letters. 

I will add, however, that it's a little fuzzy to me what should happen when apps are internationalized. In a French computer, for example, I think `ctrl + t` opens a new browser tab, even though the word for tab in French, which is "onglet", does not begin with a "t". But it's weird to me that keyboard shortcuts that were chosen because of a first-letter mnemonic in English (for example: `ctrl+s` because **S**ave begins with the letter "s") should stay the same across languages.

But the bottom line in our case is that if we are showing the keyboard shortcut in the search bar across all languages as `ctrl + k` then we should bind to the letter, not the key.

Read [event.code and event.key](https://javascript.info/keyboard-events#event-code-and-event-key) for more explanation.

I think leaving the code without a comment will actually be less confusing than adding a somewhat cryptic comment about key-mapping because standard practice is to use `event.key` for most keyboard shortcuts.